lm75bd/lm75bd.c

```diff
@@ -27,8 +27,43 @@ error_code_t lm75bdInit(lm75bd_config_t *config) {
 }
 
 error_code_t readTempLM75BD(uint8_t devAddr, float *temp) {
-  /* Implement this driver function */
+  uint8_t bufRead[1];
+  uint8_t bufWrite[2];
+  bufRead[0] = 0;
+  if (temp == NULL)
+  {
+    return ERR_CODE_INVALID_ARG;
+  }
```

Comment:
generally best to validate the parameters at the top of your function before you do anything else